packages/ppx/src/declarations_to_emotion.re

```diff
@@ -324,6 +328,23 @@ let rec render_function_calc = (~loc, calc_sum) => {
     };
   };
 }
+and render_function_min_or_max = (~loc, calc_sums: list(Types.calc_sum)) => {
+  switch (calc_sums) {
+  | [] => raise(Invalid_value("expected at least one argument"))
+  | [x, ...xs] =>
+    let calc_sums =
+      [x] |> List.append(xs |> List.map(x => x) |> List.rev) |> List.rev;
```

Comment:
Remove `List.map(x => x)` and simplify this expression please

Reply:
Improved in https://github.com/davesnx/styled-ppx/pull/411/commits/2d6b9a6c4396737961d6d0f9d9819ebd9d34ec90